draft-ietf-acme-acme.md

```diff
@@ -124,8 +124,8 @@ no out-of-band human intervention.
 
 Prior to ACME, when deploying an HTTPS server, an operator typically gets a
 prompt to generate a self-signed certificate.  If the operator were instead
-deploying an ACME-compatible web server, the experience would be something like
-this:
+deploying an HTTPS server using an ACME client, the experience would
```

Comment:
I think we should be agnostic to "HTTPS server + ACME client" (like Certbot) vs "HTTPS server with built-in ACME client" (like Caddy), especially since we'd like more HTTPS servers to build in automatic ACME support.